### DISCUSSION TITLE
Export single branch

### DISCUSSION THREAD

**Discussion by swoop124** [dani-garcia/vaultwarden#1737] [Q&A] [answered]
hello,

i am using Vaultwardon for some time now. It is great. I can use Foders and Subfolders. Now i have the challenge to export one single branch and all the logins i have in this branch.
How do i do that, because i can only export the whole passwordsafe.

btw.
when i export the whole safe and edit the csv, it seams there are not all saved password there.

thanks

**Comment by swoop124** [comment]
Hello,

ok, i found out, when i share passwords with one or more collections, than the passwords are not been exported.
Only the passwords without a collection are exported. But how can i export them too, or how can i delete the shares?
When i try to exclude a collection from a password, i get an errormessage that tells me, that i have to have one collection selected.

**Comment by jjlin** [comment, marked as answer]
> i am using Vaultwardon for some time now. It is great. I can use Foders and Subfolders. Now i have the challenge to export one single branch and all the logins i have in this branch.
> How do i do that, because i can only export the whole passwordsafe.

Exports are implemented in the client, and the Bitwarden clients don't support partial exports. If you think this is an important feature for some reason, you can make your case at https://community.bitwarden.com/c/feature-requests/5/l/latest.

> btw.
> when i export the whole safe and edit the csv, it seams there are not all saved password there.

Entries owned by an organization need to be exported separately. See https://bitwarden.com/help/article/export-your-data.

**Comment by swoop124** [comment]
thank you.
i worked great.

But the only thing i don't understand is. Why can't i delete a password from a collection.

**Reply by jjlin** [reply]
An org entry needs to be in at least one collection, otherwise it's not very useful.
